addOns/reports/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed 
```

Comment:
There's already a `Changed` section although I would mention this under `Added`, since it's a new "field".
Also good to mention under `Deprecated` section that the "old" other info entry is to be phased out.

Reply:
okk, please tell me how to write tell in more details.